advanced/Scripts/list.sh

```diff
@@ -138,7 +140,7 @@ AddDomain() {
       if [[ "${verbose}" == true ]]; then
       echo -e "  ${INFO} Adding $1 to wildcard blacklist..."
       fi
-      reload=true
+      reload="restart"
```

Comment:
Changing it from true to restart gives the variable a second purpose - we pass it through to `restartDNS` so that a wildcard update restarts `dnsmasq` instead of defaulting to "force-reload" (which in actual fact is just a `SIGHUP`).